casr/src/bin/casr-afl.rs

```diff
@@ -275,16 +275,22 @@ fn main() -> Result<()> {
     info!("Using {} threads", num_of_threads - 1);
     let counter = RwLock::new(0_usize);
     let total = crashes.len();
-    custom_pool.join(
-        || {
-            let _ = crashes.par_iter().try_for_each(|(_, crash)| {
-                crash.run_casr(output_dir.as_path(), timeout)?;
-                *counter.write().unwrap() += 1;
-                Ok::<(), anyhow::Error>(())
-            });
-        },
-        || util::log_progress(&counter, total),
-    );
+    custom_pool
+        .join(
+            || {
+                crashes.par_iter().try_for_each(|(_, crash)| {
+                    let Ok(_) = crash.run_casr(output_dir.as_path(), timeout) else {
```

Comment:
if let Err(e) = ... {
   bail!( Print error message here

Reply:
the same for casr-libfuzzer

Reply:
Done